src/Tools/dotnet-monitor/Extensibility/ExtensionManifest.cs

```diff
@@ -47,7 +53,11 @@ public static ExtensionManifest FromPath(string path)
 
             try
             {
-                return JsonSerializer.Deserialize<ExtensionManifest>(stream);
+                JsonSerializerOptions serializerOptions = new()
```

Comment:
nit: Can you make this a `static readonly` on the class so it only has to be create once?